### ISSUE TITLE
Infinite speech output after NVDA+b in Windows dialogs

### ISSUE THREAD

**Issue by cyrmax** [nvaccess/nvda#15219]
### Steps to reproduce:
* Open any Windows dialog (in my case it was harddrive error checking progress dialog);
* Press NVDA+b to read all content inside a window;
* Listen and wait.

### Actual behavior:
NVDA infinitely reads a part of content, in my case it repeats this:
Scanning and repairing...
progress bar  0%
Scanning and repairing...
progress bar  0%
Scanning and repairing...
progress bar  0%
And it never stops. First time i had waited for 10 minutes or so.

### Expected behavior:
NVDA should read window's content one time and stop speaking. No repeating phrases should be outputed, of course if no repeating messages are inside a window.

### NVDA logs, crash dumps and other attachments:
Here are NVDA Speech Viewer output and NVDA log in debug mode.
* [nvda-debug-log.txt](https://github.com/nvaccess/nvda/files/12207646/nvda-debug-log.txt);
* [NVDA-speech-viewer-output.txt](https://github.com/nvaccess/nvda/files/12207647/NVDA-speech-viewer-output.txt);

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2023.1.

#### Windows version:
Windows 11 22H2 (AMD64) build 22621.1992
Windows 11 pro.

#### Name and version of other software in use when reproducing the issue:
Not applicable. Using Windows built-in feature.

#### Other information about your system:
None.

#### Does the issue still occur after restarting your computer?
Yes. It can be reproduced always.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I remember that this issue exists since NVDA 2022 or even later.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

**Comment by Adriani90** [comment]
Closing as duplicate of #13597
